packages/release/proxy-env

```diff
@@ -0,0 +1,6 @@
+{{#if settings.network.http-proxy}}HTTP_PROXY={{settings.network.http-proxy}}
+http_proxy={{settings.network.http-proxy}}{{/if}}
+{{#if settings.network.https-proxy}}HTTPS_PROXY={{settings.network.https-proxy}}
+https_proxy={{settings.network.https-proxy}}{{/if}}
+NO_PROXY={{#each settings.network.no-proxy}}{{#if @index}},{{/if}}{{this}}{{/each}}{{#if settings.kubernetes.api-server}}{{#if settings.network.no-proxy}},{{/if}}{{host-and-port settings.kubernetes.api-server}},.svc,.default,.local,.cluster.local{{/if}}
```

Comment:
I don't think we need to care about port for no-proxying the api-server.  Can we simplify and have a `host` function instead?